<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Testbank</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(10,10,10);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">padding-top:20px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0" rowspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">padding-bottom:10px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="none">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
height:200px;
}</string>
          </property>
          <property name="text">
           <string>هیچکدام نیستم</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="student">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
height:200px;
}</string>
          </property>
          <property name="text">
           <string>دانشجو هستم</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font-family:Aviny;
font-size:44px;
color:white;</string>
          </property>
          <property name="text">
           <string>تستبانک مرجع سوالات آزمونی</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="teacher">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
height:200px;
}</string>
          </property>
          <property name="text">
           <string>مدرس هستم</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="both">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
height:200px;
}</string>
          </property>
          <property name="text">
           <string>هردو هستم</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3" rowspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="2">
         <widget class="QLabel" name="label_15">
          <property name="styleSheet">
           <string notr="true">padding-left:250px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="enter">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
}</string>
          </property>
          <property name="text">
           <string>ورود</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_13">
          <property name="styleSheet">
           <string notr="true">padding-bottom:40px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="username_enter">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
}</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>نام کاربری</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">font-family:Aviny;
font-size:44px;
color:white;</string>
          </property>
          <property name="text">
           <string>ورود به حساب کاربری</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="styleSheet">
           <string notr="true">padding-left:250px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="password_enter">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
}</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>رمز عبور</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="back">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
}</string>
          </property>
          <property name="text">
           <string>بازگشت</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="signup">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
}</string>
          </property>
          <property name="text">
           <string>ایجاد حساب</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_16">
          <property name="styleSheet">
           <string notr="true">font-family:Aviny;
font-size:20px;
color:white;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="6" column="1">
         <widget class="QPushButton" name="create">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
}</string>
          </property>
          <property name="text">
           <string>ایجاد</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="password_create">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
}</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>رمز عبور</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_28">
          <property name="styleSheet">
           <string notr="true">font-family:Aviny;
font-size:44px;
color:white;</string>
          </property>
          <property name="text">
           <string>ایجاد حساب کاربری</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="email_create">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
}</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>ایمیل</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="username_create">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
}</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>نام کاربری</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_30">
          <property name="styleSheet">
           <string notr="true">padding-left:250px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QPushButton" name="signin">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
}</string>
          </property>
          <property name="text">
           <string>ورود به حساب</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_31">
          <property name="styleSheet">
           <string notr="true">padding-left:250px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QPushButton" name="back1">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:34px;
color:white;
}</string>
          </property>
          <property name="text">
           <string>بازگشت</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="label_33">
          <property name="styleSheet">
           <string notr="true">padding-bottom:40px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_8">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="1" colspan="2">
         <widget class="QLabel" name="label_26">
          <property name="styleSheet">
           <string notr="true">padding-bottom:10px;
background-color:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_27">
          <property name="styleSheet">
           <string notr="true">padding-top:10px;
padding-bottom:10px;
background-color:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QLabel" name="label_25">
          <property name="styleSheet">
           <string notr="true">padding-bottom:20px;
background-color:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="question">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:30px;
color:white;
padding-bottom:5px;
padding-top:5px;
}</string>
          </property>
          <property name="text">
           <string>افزدون سوال</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_40">
          <property name="styleSheet">
           <string notr="true">padding-top:10px;
padding-bottom:10px;
background-color:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" rowspan="2">
         <widget class="QLabel" name="label_36">
          <property name="styleSheet">
           <string notr="true">padding-left:220px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_37">
          <property name="styleSheet">
           <string notr="true">padding-left:220px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="back2">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:30px;
color:white;
padding-top:5px;
padding-bottom:5px;
}</string>
          </property>
          <property name="text">
           <string>بازگشت</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QListWidget" name="q_list">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0,255,76);
font-family:Aviny;
font-size:34px;
border-radius:10px;
color:white;
text-align:center;</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="layoutMode">
           <enum>QListView::Batched</enum>
          </property>
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2" rowspan="3">
         <widget class="QLabel" name="label_35">
          <property name="styleSheet">
           <string notr="true">padding-bottom:10px;
background-color:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_34">
          <property name="styleSheet">
           <string notr="true">background-color:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="2" rowspan="6">
         <widget class="QLabel" name="label_71">
          <property name="styleSheet">
           <string notr="true">background:transparent;
padding-right:200px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="q_grade">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:30px;
color:white;
padding-bottom:5px;
padding-top:5px;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>پایه تحصیلی</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="q_lesson">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:30px;
color:white;
padding-bottom:5px;
padding-top:5px;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>درس مورد نظر</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="6">
         <widget class="QLabel" name="label_44">
          <property name="styleSheet">
           <string notr="true">background:transparent;
padding-left:200px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_41">
          <property name="styleSheet">
           <string notr="true">font-family:Aviny;
font-size:44px;
color:white;</string>
          </property>
          <property name="text">
           <string>سوال مورد نظر رو طرح کن</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="q_title">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:30px;
color:white;
padding-bottom:5px;
padding-top:5px;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>صورت سوال</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="save_question">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:30px;
color:white;
padding-bottom:5px;
padding-top:5px;
}</string>
          </property>
          <property name="text">
           <string>افزدون سوال</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_42">
          <property name="styleSheet">
           <string notr="true">background:transparent;
padding-bottom:10px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_43">
          <property name="styleSheet">
           <string notr="true">background:transparent;
padding-top:10px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="q_answer">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 255, 76);
border-radius:10px;
font-family:Aviny;
font-size:30px;
color:white;
padding-bottom:5px;
padding-top:5px;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>پاسخ سوال</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="fill_all">
          <property name="styleSheet">
           <string notr="true">font-family:Aviny;
font-size:20px;
color:white;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
